manager/aica_django/microagents/online_learning.py

```diff
@@ -44,6 +44,8 @@ def periodic_trainer(period_seconds: int = 300) -> NoReturn:
     while True:
         bad_traffic, _, _ = graph_db.graph.execute_query(bad_traffic_query)
         good_traffic, _, _ = graph_db.graph.execute_query(good_traffic_query)
+        logger.warning(f"Bad traffic length {len(bad_traffic)}")
```

Comment:
Probably logging at the info level versus warning would be appropriate for this and the following line